src/Containers/List/List.tsx

```diff
@@ -16,7 +16,6 @@ export interface ListProps extends React.HTMLAttributes<HTMLDivElement> {
     left?: string;
     onCloseTranslateXAxis?: string;
     cssPosition?: string;
-    isToggleable?: boolean;
```

Comment:
Why remove?

Reply:
Maybe isFrozen or isHideable

Reply:
It serves no purpose, functionality is effectively replaced by toggleComponent. Pass setIsOpen to either togglecomponent or to listItem to control toggled state.